src/internal/mediaUtil.ts

```diff
@@ -74,3 +74,25 @@ export function validateViewImagesInput(uriList: media.ImageUri[]): boolean {
   }
   return true;
 }
+
+/**
+ * Returns true if the scan barcode param is valid and false otherwise
+ */
+export function validateScanBarCodeInput(barCodeConfig: media.BarCodeConfig): boolean {
+  if (barCodeConfig != null) {
+    if (
+      barCodeConfig.barCodeType != null &&
+      (barCodeConfig.barCodeType < media.BarCodeType.All ||
+        barCodeConfig.barCodeType > media.BarCodeType.TwoDimensional)
+    ) {
+      return false;
+    }
+    if (
```

Comment:
The opening and closing braces both ( ) and { } have been formatted as per c/c++ style. Javascript's ECMA script specification recommends that it be formatted like this: https://stackoverflow.com/questions/3960518/javascript-formatting-must-braces-be-on-the-same-line-as-the-if-function-etc-ke

Reply:
This is formatted as per lint rules specified in Teams JS SDK repo